expression/builtin_op.go

```diff
@@ -353,7 +356,15 @@ func (b *builtinUnaryOpSig) eval(row []types.Datum) (d types.Datum, err error) {
 		case types.KindInt64:
 			d.SetInt64(-aDatum.GetInt64())
```

Comment:
I guess opcode.Minus operation on `math.MinInt64` would  get wrong result as range of negative is greater then range of positive by 1.